### DISCUSSION TITLE
Renovate eating up a lot of disk space about once per month

### DISCUSSION THREAD

**Discussion by LVerneyPEReN** [renovatebot/renovate#11337] [Request Help]
### How are you running Renovate?

Self-hosted

### Please select which platform you are using if self-hosting.

gitlab.com

### If you're self-hosting Renovate, tell us what version of Renovate you run.

renovate/renovate:25.56.9@sha256:38d3f087a89047301fdf12528cf66975fc092a4465f7a34b87575eaad003c9c9

### Describe the bug

Hi,

I noticed that about once a month, Renovate is taking a lot more disk space than expected. On the disk space usage capture below, Renovate running every night at 4am is noticeable (small bump in the graph), but for some reason it is taking a lot of disk space on August, 19th.

![image](https://user-images.githubusercontent.com/58298410/130052262-a8407e53-9877-417c-b4d1-6ca78f21609f.png)

I had similar behavior on June, 24th and July, 19th. When this happens, my Gitlab-CI runner runs out of disk space, resulting in

```
ERROR: Disk space error - skipping (repository=xxx)
Error: ENOSPC: no space left on device, write
```

in Renovate CI job and a fail of the CI.

This disk space is eaten by many uncleaned Docker volumes, which I have to `docker volume prune` to reclaim space (about 30GB).

Sadly, comparing the logs of the failing run (today) and the previous runs, I do not see any relevant ideas to explain this unusual behavior.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste any log here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by viceice** [comment]
Are you using our official gitlab pipeline templates ? https://gitlab.com/renovate-bot/renovate-runner

They had a cache bug, which was fixed in v1.68.0 and v1.68.1.
